packages/flutter/lib/src/material/tab_bar_theme.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
```

Comment:
You shouldn't need to import this, import widgets.dart instead.

Reply:
I think it was actually an unused import, not sure when I added that. Removed.